backend/model/transactions.py

```diff
@@ -51,3 +55,16 @@ def update_from_db(self, data):
     @classmethod
     def find_by_id(cls, id_transaction):
         return cls.query.filter_by(id_transaction=id_transaction).first()
+
+    @classmethod
+    def send_confirmation_mail(cls, transaction):
```

Comment:
Fes que el metode sigui del format `(emailDestination, subject, message) ` aixi es pot utilitzar des de una altre clase. A mes estaria be, no se si ja ho fa, que llançi una Excepcio si no s'envia el correu be.

Reply:
Si el correu no es pot enviar perquè ha fallat alguna cosa sí que surt excepció. En canvi quan no existeix el mail receptor simpolement rebem un mail a bookshelterES@gmail.com indicant que no existeix el mail. 
Això es solucionarà més endavant quan fiquem la verificació de mail.

Reply:
O sea en produir-se un error de correu s'ens envia un correu a nosaltres avisant que no existeix. Aixo et refreixes a que ho fas tu o ho fa gmail automaticament

Reply:
Ho fa gmail automàticament